src/tests/testRandomUuid.cc

```diff
@@ -78,15 +78,7 @@ void
 TestRandomUuid::testInvalidIds()
 {
     for (const auto &id: InvalidIds) {
-        try {
-            RandomUuid uuid(id.second);
-            std::cerr << std::endl
-                      << "FAIL: " << id.first
-                      << Debug::Extra << "error: should be rejected" << std::endl;
-        } catch (const TextException &e) {
-            continue; // success, caught a malformed UUID
-        }
-        CPPUNIT_FAIL("failed to reject an invalid UUID");
+        CPPUNIT_ASSERT_THROW(RandomUuid uuid(id.second), TextException);
```

Comment:
I am a bit worried that `uuid` is now seemingly unused, but since compilers do not seem to think so, I am OK with the current PR code.

FWIW, I would prefer something more explicit like the code below, but I do not know whether it would work.

```suggestion
        CPPUNIT_ASSERT_THROW((void)RandomUuid(id.second), TextException);
```

Reply:
I expect the compilers are smart enough to notice that the `RandomUuid` has external side effects on the shared/static RNG engine.